h2o-core/src/main/java/water/fvec/RollupStats.java

```diff
@@ -288,12 +289,15 @@ static void start(final Vec vec, Futures fs, boolean computeHisto) {
     if( vec.isString() ) computeHisto = false; // No histogram for string columns
     final Key rskey = vec.rollupStatsKey();
     RollupStats rs = getOrNull(vec,rskey);
-    if(rs == null || (computeHisto && !rs.hasHisto()))
-      fs.add(new RPC(rskey.home_node(),new ComputeRollupsTask(vec,computeHisto)).addCompleter(new H2OCallback() {
-        @Override public void callback(H2OCountedCompleter h2OCountedCompleter) {
+    if(rs == null || (computeHisto && !rs.hasHisto())) {
+      Log.debug("Will calculate RollupStats, key = ", rskey);
```

Comment:
Yeah, I am not sure about that either :) Maybe trace would be better but we can easily changed it to trace in the future if someone complains.